src/DynamicData.Tests/Binding/BindingListToChangeSetFixture.cs

```diff
@@ -1,8 +1,8 @@
-﻿using DynamicData.Binding;
-using FluentAssertions;
-using System;
+﻿using System;
```

Comment:
Excellent and encouraging to see loads of unit tests

Reply:
Gotta love those unit tests! :)